src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -146,6 +146,215 @@ <h2 class="panel-title text-center">Hypotheses Evaluating</h2>
+          <div class="flex-grow-1 flex-container in-column-direction">
+            <h2 class="card-pf-title card-pf-utilization-details">
+              <span>My Work items</span>
+              <span class="badge">20</span>
+            </h2>
+            <div class="analyze-overview-add-items">
+              <div class="card-pf-body text-center">
+                <i class="pficon pficon-add-circle-o"></i>
+                <p class="card-pf-utilization-lg-p">Create Work-items</p>
+                <p class="text-center">Planner will help you to create different
+                  <br /> types of work-items, allows you to assign a team
+                  <br />members and iterations</p>
+                <button class="btn btn-primary btn-lg margin-top-15">Create a work-item</button>
+              </div>
+            </div>
+            <div class="analyze-overview-items-detail">
+              <div class="card-pf-body margin-0">
+                <ul>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                  <li>
+                    Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wirframe label
+                  </li>
+                </ul>
+              </div>
+              <div class="card-pf-footer padding-bottom-15">
+                <a class="margin-bottom-5">
+                  <i class="pficon pficon-add-circle-o"></i>
+                  <span>Add work item</span>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </aside>
+    </div>
+    <div class="flex-grow-4 " >
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-1 flex-container in-column-direction">
+            <h2 class="card-pf-title card-pf-utilization-details">
+              <span>Codebase</span>
+              <span class="badge">20</span>
+            </h2>
+            <div class="analyze-overview-add-items">
+              <div class="card-pf-body text-center">
+                <i class="pficon pficon-add-circle-o"></i>
+                <p class="card-pf-utilization-lg-p">Create Work-items</p>
```

Comment:
This should be 'Create Work Item', with Work Item as two words, each capitalized.